report/id.go

```diff
@@ -1,8 +1,13 @@
 package report
 
 import (
+	"hash"
+	"hash/fnv"
 	"net"
 	"strings"
+	"sync"
+
+	cache "github.com/hashicorp/golang-lru"
```

Comment:
The package name differs from the last part of the URL (see https://github.com/hashicorp/golang-lru/blob/master/lru.go#L4) and I thought this made it more obvious.